app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count = 0
+    without_mask = 0
+    without_vaccine = 0
+    for friend in friends:
+        try:
+            Cafe.visit_cafe(cafe, friend)
```

Comment:
The `visit_cafe` method should be called on the `cafe` instance, not the `Cafe` class. Change `Cafe.visit_cafe(cafe, friend)` to `cafe.visit_cafe(friend)` to correctly call the instance method.